docs/configuration/configuration.rst

```diff
@@ -4,10 +4,262 @@
+
+The observatory model has in internal object that stores the state of the observatory.
+In parallel, the Scheduler also keeps a separate copy of the observatory state.
+
+The Scheduler copy of the observatory state is kept in synchronization with the actual observatory state, by listening for the state of the different components and continually updating this object.
+
+When the Scheduler is updating the telemetry stream, it will synchronize the observatory state in the model with its observatory state.
+From that on, the Driver can modify the state of the observatory model to compute future observations.
+
+The following is a list of available models.
+Each model has a specific section in the configuration for inputting their parameters.
+The parameters for each model can be found in the schema file.
+
+* location; The location of the observatory.
+* observatory_model; The observatory model.
+* sky; Sky model, including sky brighness models.
+* seeing; Seeing model, includes functionality to convert from raw seeing measurements into seeing in the different bands and at different elevations.
+
+The models all come with suitable default configurations so users will seldom require changing them.
+One example of updating some of the observatory model configuration is as follows;
+
+.. code-block:: yaml
+
+   models:
+       observatory_model:
+           camera:
+               filter_max_changes_burst_num: 1
+               filter_max_changes_avg_num: 30000
+           optics_loop_corr:
+               tel_optics_cl_alt_limit:
+               - 0
+               - 30
+               - 90
+
+In the case above, the user is customizing the values of ``filter_max_changes_burst_num`` and ``filter_max_changes_avg_num`` in the ``camera`` submodule and ``tel_optics_cl_alt_limit`` of the ``optics_loop_corr`` submodule of the ``observatory_model``.
+
+.. _Configuration_details_telemetry_parameters:
+
+Telemetry Parameters
+====================
+
+The Scheduler exposes a pretty rich interface for users to define how it interacts with the telemetry stream.
+In short, users can select a table in the EFD to be queried along with some basic information about how the Scheduler will query.
+
+The telemetry configuration consists of the following parameters:
+
+* efd_name; the name of the EFD instance which will be queried, e.g., summit or efd.
+* streams; List of telemetry streams.
+
+  Each item have the following properties:
+
+  * name: Name of the telemetry.
+  
+    This is basically a dictionary key the Scheduler will use to identify the telemetry.
+  
+  * efd_table: The name of the EFD table to query the information from.
+  * efd_columns: The column in the EFD table to query the information from.
+  * efd_delta_time: Length of history to request from the EFD (in seconds).
+  * fill_value: Which value to assign the telemetry when no data point is obtained.
+
+An example telemetry configuration is as follows:
```

Comment:
The example items do not have a `fill_value`. Either please add them or explain that the `fill_value` is not mandatory.